Add email variants offering mentoring to all teams (#78)
srobo/team-emails: SR2021/2021-02-14-league-2-followup.md

---

> to: Teams who attended the leagues but aren't very active

Reviewer: ```suggestion
to: Teams who attended the leagues and are active
```

> subject: Preparing for league 3

Author: I'm actually not that happy with this subject line, would be great to have some other ideas.
I wanted to avoid it seeming to focus on support, because these teams are doing fine and we don't want to encourage them too much to ask for a mentor.

Reviewer: Support is basically the entire content of this email, though. If that's _not_ the focus, I don't know what is.

We could put some reminders about the next league, friendlies etc, but then we should also add to the other emails.

Author: @WillB97 said (on the other thread):
> Probably something more like "Need extra support?" over directly mentioning mentors in the subject.

These teams are already doing fairly well, which was why I was trying to avoid the framing being around them needing support. If we had more mentors it could be something like "Mentors now available" (framing it as news), but given that we don't have many mentors that might be misleading.

Author: Now that we have the friendlies in here, this subject makes a lot more sense, so I think we're good now :)

Reviewer 2: Can we go with something like "Just checking in" or is that way too informal

> subject: Preparing for league 3

Reviewer: But mah unicode

```suggestion
subject: Preparing for league Ⅲ
```

Author: Sooo, the league sessions are on the website with Arabic numerals. It's the rules' modules which use the roman numerals. I'm therefore tempted to leave this as is.

Reviewer 3: Probably something more like "Need extra support?" over directly mentioning mentors in the subject.

---

Hi,

It was great to see your robot at the second league session. We hope you found
it as exciting as we did and are looking forward to the new challenges which
module Ⅲ brings. The full rulebook and simulator updates are available on our
website, so do check them out if you haven't already!

We just wanted to check on how you're doing and whether there's any more help
you need?

As a reminder, there's usually lots of volunteers around in Discord to help out
if you need anything, whether it's help programming your robot, organising your
team or you just need a fresh pair of eyes on something. Our volunteers have a
wealth of experience from both competing in Student Robotics and helping lots of
teams, so there's a good chance they can help :)

We also have a limited number of volunteers who would be able to regularly
mentor your team. This would provide you some more hands-on support in the form
of a volunteer who is able to be around whenever you regularly meet and would
get to know your team and your robot. Whatever platform you use (whether it's
Discord or something else), they'd be able to join your meetings to help you

> out. (For safeguarding reasons we need your team leader to also be in any video
> calls; your team leader will have details about this).

Reviewer: ```suggestion
out. (For safeguarding reasons we need you to also be in any video calls).
```

Author: c6bca27 fixes this for all three emails.

---
to: Teams who attended the leagues and are active
subject: Preparing for league 3
---

Hi,

It was great to see your robot at the second league session. We hope you found
it as exciting as we did and are looking forward to the new challenges which

> module Ⅲ brings. The full rulebook and simulator updates are available on our

Reviewer 2: Similar to my other comment should we inline these links?

Author: d9242d7e36b95c8a715e80bf895478aa4909413d
